### ISSUE TITLE
mAP of object detection

### ISSUE THREAD

**Issue by feanor21115** [jakc4103/DFQ#19]
Dear jakc,

Thank you for sharing your work of DFQ.
Very appreciate it!!

However, I have found that
the mAP of mobilnetv2 SSD_lite that https://github.com/qfgaohao/pytorch-ssd had claimed is 68.607.
And your result of original FP32 is 60.69.

Is there any reason that causes the phenomenon?
The fusion of batch-normal or something else?

Thank you.

**Comment by jakc4103** [comment]
Basically batchnorm fusion won't change the result in FP32 precision. The reason is probably evaluation metric.

**Comment by feanor21115** [comment]
But, with the same evaluation metric,
if the "create model part" and "create predictor part" have been merged like that

elif args.net == 'mb2-ssd-lite':
        net = create_mobilenetv2_ssd_lite(.....)
        net.load(args.trained_model)
        predictor = create_mobilenetv2_ssd_lite_predictor(.....)

net.to('cpu')   

I will obtain the mAP of 68.7.
Is my trick reasonable?

**Comment by jakc4103** [comment]
The thing is that I forgot to add net.eval() after the model is loaded. At [HERE](https://github.com/jakc4103/DFQ/blob/master/main_ssd.py#L212), it actually do a forward call to get the model graph in training mode, which cause the batchnorm stats dirty.

The latest commit should fix this, I will update the corrected mAP as I finish all the evaluation.
Great job spotting this :+1:

**Comment by feanor21115** [comment]
Thank you, I will try it.

All the best!!
